packages/driver/test/cypress/integration/commands/connectors_spec.coffee

```diff
@@ -752,21 +751,40 @@ describe "src/cy/commands/connectors", ->
 
           cy.wrap(obj).its("foo.bar.baz").should("eq", "baz")
 
-        it "throws when property does not exist on the subject", (done) ->
-          cy.on "fail", (err) =>
-            lastLog = @lastLog
+        it "does not throw when property does not exist on the subject", ->
+          cy.noop({}).its("foo")
 
-            expect(err.message).to.include "cy.its() errored because the property: 'foo' does not exist on your subject."
-            expect(lastLog.get("error").message).to.include(err.message)
+        it "retries when yielded undefined value", ->
+          obj = {}
+
+          obj.foo = undefined
+
+          setTimeout ()->
```

Comment:
using `setTimeout` here can lead to possibly non-deterministic behavior...

there is a safer and better way to do this...

```js
cy.on('retry', _.once(() => {
  obj.foo = true
})
```

this ensure the assertion retries once, which then allows you to set the property

Reply:
:+1: